doc/classes/ProjectSettings.xml

```diff
@@ -235,6 +235,16 @@
 		</method>
 	</methods>
 	<members>
+		<member name="accessibility/accessibility/accessibility_support" type="int" setter="" getter="" default="0">
+			Accessibility support mode:
+			If [code]0[/code], accessibility support is fully disabled.
+			If [code]1[/code], accessibility support is enabled, and accessibility information updates are processed regardless of current assistive apps' status.
+			If [code]2[/code], accessibility support is enabled, but accessibility information updates are processed only if an assistive app (e.g. screen reader or Braille display) is active (default).
```

Comment:
Should probably mention the human readable names of the enum options.
In other places we seem to do it like this:
```
- [b]Auto[/b] ([code]0[/code]): ...
- [b]Always Active[/b] ([code]1[/code]): ...
- [b]Disabled[/b] ([code]2[/code]): ...
```